lib/skate_web/router.ex

```diff
@@ -39,7 +40,6 @@ defmodule SkateWeb.Router do
     plug :fetch_session
     plug :fetch_flash
     plug :protect_from_forgery
-    plug :put_secure_browser_headers
```

Comment:
**question:** what impact does this have on the `/api` endpoints? Those have `:browser` but not `:accepts_html`, so they will no longer get `:put_secure_browser_headers` after this change.

Reply:
Augh - I thought that was a pure refactor, but I didn't notice that this effectively removed `:put_secure_browser_headers` from `:accepts_json`.